src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -84,18 +95,15 @@ private void additionalAccountProperties(JsonObject loanRepresentation, Collecti
     write(loanRepresentation, LoanProperties.FEESANDFINES, feesAndFinesSummary);
   }
 
-  private void additionalLoanPolicyProperties(JsonObject loanRepresentation, LoanPolicy loanPolicy) {
-    if (loanPolicy == null) {
-      log.info("Unable to add loan policy properties to loan {}," + " loanPolicy is null", loanRepresentation.getString("id"));
-      return;
-    }
-    JsonObject loanPolicySummary = loanRepresentation.containsKey(LoanProperties.LOAN_POLICY)
-        ? loanRepresentation.getJsonObject(LoanProperties.LOAN_POLICY)
-        : new JsonObject();
-
-    loanPolicySummary.put("name", loanPolicy.getName());
+  private void additionalPolicyProperties(JsonObject representation,
+                                          CirculationPolicy policy,
+                                          String policyName) {
```

Comment:
Similarly to above, please change the indentation

Reply:
Changed